docs/virtual_machines/disks_and_volumes.md

```diff
@@ -264,24 +264,24 @@ datasource may look like this:
 
 Allows connecting a `PersistentVolumeClaim` to a VM disk.
 
-Use a PersistentVolumeClaim when the VirtualMachineInstance’s disk needs
-to persist after the VM terminates. This allows for the VM’s data to
+Use a PersistentVolumeClaim when the VirtualMachineInstance's disk needs
+to persist after the VM terminates. This allows for the VM's data to
 remain persistent between restarts.
 
-A `PersistentVolume` can be in “filesystem” or “block” mode:
+A `PersistentVolume` can be in "filesystem" or "block" mode:
 
 -   Filesystem: For KubeVirt to be able to consume the disk present on a
-    PersistentVolume’s filesystem, the disk must be named `disk.img` and
+    PersistentVolume's filesystem, the disk must be named `disk.img` and
     be placed in the root path of the filesystem. Currently the disk is
-    also required to be in raw format. &gt; **Important:** The
```

Comment:
unsure on this one. I'd assume markdown takes care of that?

Reply:
There was some kind of special character in this line that doesn't show up under normal text editing tools. HTMLProofer would fail with a non-ascii character error.  

The following grep pointed to this line
```
for i in `find docs/ -type f | grep ".md"`; do grep --color='auto' -P -n "[^\x00-\x7F]" $i && echo $i; done
```

I couldn't figure out where the error was.  Replacing the entire line cleared the grep